pkg/connector/users.go

```diff
@@ -62,11 +67,26 @@ func (u *userBuilder) List(ctx context.Context, parentResourceID *v2.ResourceId,
 		return nil, "", nil, err
 	}
 
-	users, nextCursor, err := u.client.ListUsers(ctx, bag.PageToken())
+	domains, _, err := u.client.ListDomains(ctx, bag.PageToken())
```

Comment:
I like this change. I'm just worried that we might have a customer with multiple authentication domains. It'd be nice if it fell back to the old behavior if the number of domains was not equal to 1.

Reply:
Ok, Fixed